packages/manager/modules/cloud-connect/src/details/statistics/statistics.controller.js

```diff
@@ -0,0 +1,216 @@
+import get from 'lodash/get';
+import map from 'lodash/map';
+
+import {
+  PERIOD,
+  PERIOD_ENUM,
+  STATISTICS,
+  TYPE,
+  TYPE_ENUM,
+  TYPE_LABELS,
+} from './statistics.constant';
+
+export default class CloudConnectStatisticsCtrl {
+  /* @ngInject */
+  constructor($q, $translate, CucCloudMessage, cloudConnectService) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.cloudConnectService = cloudConnectService;
+    this.PERIOD = PERIOD;
+    this.STATISTICS = STATISTICS;
+    this.TYPE = TYPE;
+    this.TYPE_LABELS = TYPE_LABELS;
+  }
+
+  $onInit() {
+    this.periodOptions = PERIOD_ENUM;
+    this.typeOptions = TYPE_ENUM;
+
+    this.graphType = this.TYPE.TRAFFIC;
+    this.graphPeriod = this.PERIOD.DAILY;
+
+    this.trafficLoader = {
+      series: [],
+      data: [],
+    };
+
+    this.displayedGraph = this.TYPE.TRAFFIC;
+    this.colors = this.STATISTICS.colors;
+    this.options = angular.copy(this.STATISTICS.options);
```

Comment:
```suggestion
    this.options = {...this.STATISTICS.options};
```
not doing the same ?